README.markdown

```diff
@@ -296,15 +274,6 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
```

Comment:
I am neutral on this. The example above is certainly compelling, but like it feels it is exaggerating our situation. It is probably more like

```
it makes reading slower
   like this because we're used to
   having multiple words
   on a single line.
```

and that is not that hard to read.